psycopg/psycopg/connection_async.py

```diff
@@ -90,6 +91,20 @@ async def connect(
         row_factory: Optional[AsyncRowFactory[Row]] = None,
         **kwargs: Any,
     ) -> "AsyncConnection[Any]":
+
+        if sys.platform == "win32":
+            loop = asyncio.get_running_loop()
+            if isinstance(loop, asyncio.ProactorEventLoop):
+                # TODO: is there a general psycopg error not related
```

Comment:
Probably `InterfaceError` is good, as it's not a `DatabaseError` (see [inheritance tree](https://www.psycopg.org/psycopg3/docs/api/module.html))